param/motion_planner.yaml

```diff
@@ -52,4 +52,22 @@ k_position_y: 1.25
 # p-term for z height for position holder task
 k_position_z: .90
 # maximum radial distance from target point at which hold position task is allowed to run
-max_holding_range: 1
+max_holding_range: 1.0
```

Comment:
In this case it might be easiest to just make a fix commit to delete the extra parameters before it gets merged. Aaron might know a better way though.